.github/workflows/ci.yml

```diff
@@ -168,9 +170,9 @@ jobs:
     steps:
       - name: checkout
         uses: actions/checkout@v2
-      - name: sync+blocks remove node test
-        run: make dockertest-sync-blocks-remove-node
```

Comment:
This one too, was it removed or replaced? I don't see it anywhere else

Reply:
you can see them run separately